### PR TITLE
Crash due to non-existing empty string

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: #946,  https://help.nextcloud.com/t/nextcloud-dev-doesnt-start/12389

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
DroneCI keeps failing:
```
found errors: 118
found warnings: 852
previous errors: 74
previous warnings: 859
FAIL: error count increased
```

**Comment by tobiasKaminsky** [comment]
This is due to the fact that I have set "string empty" to "translatable=false" to avoid all transifex translators doing unnecessary steps. But, for some (43 language files) there was already a "string empty).

So I will remove it now in those files as it is unneeded there as android will always fall back to the values/strings.xml file and there "string empty" is declared.

File: build.gradle [hunk 1]
```diff
@@ -195,7 +195,7 @@ dependencies {
     compile 'com.googlecode.ez-vcard:ez-vcard:0.10.2'
 
     // uncomment for gplay, modified
-    //compile 'com.google.android.gms:play-services:10.2.4'
+    compile 'com.google.android.gms:play-services:10.2.4'
```
Comment (AndyScherzinger) [inline build.gradle]: @tobiasKaminsky it seems you accidentally activated GMS

File: build.gradle [hunk 2]
```diff
@@ -239,4 +239,4 @@ tasks.withType(Test) {
 }
 
 // uncomment for gplay, modified (must be at the bottom)
-//apply plugin: 'com.google.gms.google-services'
\ No newline at end of file
+apply plugin: 'com.google.gms.google-services'
```
Comment (AndyScherzinger) [inline build.gradle]: @tobiasKaminsky same here: it seems you accidentally activated GMS

File: build.gradle (hunk 1 above)
Comment (tobiasKaminsky) [inline build.gradle]: Damnit.

File: build.gradle (hunk 2 above)
Comment (tobiasKaminsky) [inline build.gradle]: Damnit^2

**Comment by mario** [comment]
Conflicts...

**Comment by AndyScherzinger** [comment]
👍

**Comment by mario** [comment]
@tobiasKaminsky fix conflicts so we can merge :P

**Comment by tobiasKaminsky** [comment]
@mario @AndyScherzinger please re-approve :-)

**Comment by AndyScherzinger** [comment]
👍 re-approved :)

**Comment by AndyScherzinger** [comment]
@mario can you re-approve and merge :) Pleeeease 💋

**Comment by mario** [comment]
👍

**Pull request merged**